bender.js

```diff
@@ -43,11 +43,12 @@ var config = {
     ],
 
     tests: {
-        nanospell: {
```

Comment:
It was testing boilerplate that I copied over from when I tried to make the nanospell repo test things properly.  I would like to break it into a project template (maybe yeoman) at some point, but it doesn't really make sense to do it on work time when the benefit is so little.  

The key is just used for grouping tests inside the test runner.  These tests only test the "detabler" plugin in this repo with some minimal set of default CKEditor plugins.